Formula/wget2.rb

```diff
@@ -0,0 +1,26 @@
+class Wget2 < Formula
+  desc "Multithreaded metalink/file/website downloader"
+  homepage "https://gitlab.com/gnuwget/wget2/"
+  url "https://alpha.gnu.org/gnu/wget/wget2-1.99.1.tar.gz"
+  sha256 "8d9b0b0897ff9623169db47c656d09007677f8d51de5f5cd3652258aab8b4dde"
+
+  depends_on "autoconf" => :build
```

Comment:
`autoconf` and `automake` do not seem necessary here as the tarball has already been autoconfed.

Reply:
You’re right. The git repo needs them, the release tarball not anymore. Removed.